src/components/_modals/DepositModal/SommelierTab.tsx

```
@@ -402,135 +485,53 @@ export const SommelierTab = ({
     const tokenSymbol = data?.selectedToken?.symbol
     const depositAmount = data?.depositAmount
 
-    // if swap slippage is not set, use default value
-    const slippage = data?.slippage
-
-    if (!erc20Contract) return
+    let nativeDeposit = selectedToken?.symbol === "ETH"
```

ESLint

[error] 'nativeDeposit' is never reassigned. Use 'const' instead.

(prefer-const)